tests/unit/test_queries.py

```diff
@@ -0,0 +1,74 @@
+import json
+import os
+import unittest
+
+import boto3
+from botocore.exceptions import ClientError
+from opensearchpy import OpenSearch, RequestsHttpConnection, AWSV4SignerAuth
+import pytest
+import time
+from sds_in_a_box.SDSCode import queries
+from sds_in_a_box.SDSCode.opensearch_utils.action import Action
+from sds_in_a_box.SDSCode.opensearch_utils.index import Index
+from sds_in_a_box.SDSCode.opensearch_utils.document import Document
+from sds_in_a_box.SDSCode.opensearch_utils.payload import Payload
+from sds_in_a_box.SDSCode.opensearch_utils.client import Client
+
+
+@pytest.mark.network
+class TestQueries(unittest.TestCase):
+
+    def setUp(self):
+        #Opensearch client Params
+        os.environ["OS_DOMAIN"] = 'search-sds-metadata-uum2vnbdbqbnh7qnbde6t74xim.us-west-2.es.amazonaws.com'
+        os.environ["OS_PORT"] = '443'
+        os.environ["OS_INDEX"] = "test_data"
+
+        hosts = [{"host":os.environ["OS_DOMAIN"], "port":os.environ["OS_PORT"]}]
+        
+        secret_name = "OpenSearchPassword9643DC3D-uVH94BjrbF9u"
```

Comment:
We probably don't want this on the open internet or is this a test name somehow? It seems like someone could possibly piece together things here since we are using a lot of admin and hardcoded urls throughout here.

Reply:
The good thing is this is the secret location, not the actual secret value. However, I agree, this isn't ideal. Opened issue #23 as a result of this comment.